pss/crypto/crypto.go

```diff
@@ -0,0 +1,656 @@
+// Copyright 2019 The Swarm Authors
```

Comment:
This is just a moved file? No changes?

Reply:
Sorry, these changes belong to #1731. This PR shouldn't be reviewed until that one is merged.